components/rangeslider/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "coherent-gameface-rangeslider",
-  "version": "3.0.0",
```

Comment:
Isn't this a major update? Or because we kept the main functionality and only added more it is a minor?

Reply:
Yes. We kept the main functionality and it should work as before without any breaking changes. That is why I have updated the minor version.